### ISSUE TITLE
incorporate comments from dan noble

### ISSUE THREAD

**Issue by wcornwell** [traitecoevo/data_versioning#19]

**Comment by dfalster** [comment]
L3 - title: its -> it is
L38 - remove “sea” and just stick with “change”
L43 - is -> are
L75 - Could use a couple references
L87 - sentence doesn’t quite make sense, suggest rewording  from ", datasets…”
L100 - this -> these
L107 - I don’t understand “bespoke infrastructure”; may want to define or reword.
L108- I would just integrate the last two sentences
L110 - add in a “a” after “is”.
L118 - Do you need a “Results” header? 
L123 - I would just remove this sentence as I don’t think it is necessary. Plus the next sentence gets to what you want to say anyway and says sort of a similar thing. 
L126 - remove “be similarly”
L145 - 154 - I think this is a pragmatic view, but I would have a tendency not to discuss how the tools you can use for a VDD approach are likely to be "replaced in the near future”. Of course, things change, but this will not entice the people you want the paper to appeal to to adopt this sort of approach. From my experience, some of the resistance to adopting this stuff comes from programs or repositories being “replaced” at such a high rate. They just don’t feel like its worth doing. 
L211 - remove “also able” and “the”
L219 - replace “a” with “an”
L220 - "her / his” is probably more appropriate..
L229 - 235 - I found this part to be a bit difficult to grasp as you discuss how updated versions can "download the latest version”, but then in the same sentence discuss how this works seamlessly “offline”. You may want to discuss this in greater depth as I don’t see how this is possible.
L250-254 - I would do a bit of re-wording here. Maybe cut the sentence after “GitHub”. Start with: “Indeed, building…” Then start the next sentence with “We…”. 
L275 - I don’t understand what is being said here….”current structure of scientific research is very distributed…” Probably just needs some re-wording.
Figure 1 - could use a more descriptive legend. Maybe walk the reader through it a bit more. Also, I didn’t see (a) and (b) in that figure.

**Comment by dfalster** [comment]
Closed via 8b0093ddc2d2dd6dc2cdfbc122e825f6c9f27cf4
